custom_components/tuya_local/translations/en.json

```diff
@@ -195,4 +195,4 @@
             }
         }
     }
-}
+}
```

Comment:
my vscode instance removes it on saving by default and [It seems as there is no new line in your en.json as well](https://github.com/make-all/tuya-local/blob/main/custom_components/tuya_local/translations/en.json)

Reply:
I suggest to revert this file. There is a new line in the version in git, otherwise it would not detect this as a change, and the location of the error indicator on the right hand side indicates that it is the new version that is incorrect.

Reply:
alright. off-topic: I don't understand icon-priority, can you point me to how it works?

Reply:
A lower number is higher priority (ie priority number 1 overrides all others).  Where the icon is set in more than one location, and they might indicate different icons at the same time, the priority is needed to decide which one will apply.

But setting icons should be used sparingly, because it interferes with any icon changes that the user's chosen theme makes.  The icon_priority was introduced early in the development because it was a feature in the Goldair integration that this was originally based on, and predated Home Assistant themes.  Now I mostly only set icons at the entity level for switches and sensors which would benefit from a more descriptive icon for their function, and since it is only one location setting the icon, there is no need for a priority.